internal/cmd/ansible-operator/run/cmd.go

```diff
@@ -141,41 +146,31 @@ func run(cmd *cobra.Command, f *flags.Flags) {
 	// TODO: probably should expose the host & port as an environment variables
 	options = f.ToManagerOptions(options)
 	if options.NewClient == nil {
-		options.NewClient = func(cache cache.Cache, config *rest.Config, options client.Options, uncachedObjects ...client.Object) (client.Client, error) {
-			// Create the Client for Write operations.
-			c, err := client.New(config, options)
-			if err != nil {
-				return nil, err
-			}
-			return client.NewDelegatingClient(client.NewDelegatingClientInput{
-				CacheReader:       cache,
-				Client:            c,
-				UncachedObjects:   uncachedObjects,
-				CacheUnstructured: true,
-			})
-		}
+		options.NewClient = client.New
 	}
 
 	namespace, found := os.LookupEnv(k8sutil.WatchNamespaceEnvVar)
 	log = log.WithValues("Namespace", namespace)
+	var watchNamespaces []string
 	if found {
 		log.V(1).Info(fmt.Sprintf("Setting namespace with value in %s", k8sutil.WatchNamespaceEnvVar))
 		if namespace == metav1.NamespaceAll {
 			log.Info("Watching all namespaces.")
-			options.Namespace = metav1.NamespaceAll
+			watchNamespaces = []string{metav1.NamespaceAll}
 		} else {
 			if strings.Contains(namespace, ",") {
 				log.Info("Watching multiple namespaces.")
-				options.NewCache = cache.MultiNamespacedCacheBuilder(strings.Split(namespace, ","))
+				watchNamespaces = strings.Split(namespace, "")
```

Comment:
Based on the surrounding context of this function it looks like we need to split on a `,`:
```suggestion
				watchNamespaces = strings.Split(namespace, ",")
```

Reply:
https://github.com/operator-framework/ansible-operator-plugins/pull/37/commits/e7d8c5cd6a469fb34d39826d7aac22e3cc5fa08d

Reply:
Unresolving as it still looks to be missing the `,` as the separator here

Reply:
Ugh forgot to save the file. https://github.com/operator-framework/ansible-operator-plugins/pull/37/commits/cd8f4e8e7ff67fc1a5c7ed1570083ff24fe9c3cf